resources/services/rds/certificates.go

```diff
@@ -17,7 +17,7 @@ func RdsCertificates() *schema.Table {
 		Multiplex:    client.ServiceAccountRegionMultiplexer("rds"),
 		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
 		DeleteFilter: client.DeleteAccountRegionFilter,
-		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "thumbprint"}},
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "arn"}},
```

Comment:
The thing I had been trying to understand was if thumbprint could be replicated accross region, but this seems to have fixed that issue as region is part of arn

Reply:
wouldn't account_id be included in arn? we can just use that instead?

Reply:
correct, fixed

Reply:
So maybe the better PK choice is `arn` and `thumbprint`?

Reply:
Actually nevermind, remembered why we need `account_id`